RGB.NET.Devices.Wooting/Keyboard/WootingKeyboardLedMappings.cs

```diff
@@ -104,7 +104,7 @@ internal static class WootingKeyboardLedMappings
         { LedId.Keyboard_Space, (5, 6) },
         { LedId.Keyboard_RightAlt, (5, 10) },
         { LedId.Keyboard_RightGui, (5, 11) },
-        { LedId.Keyboard_Application, (5, 12) },
+        { LedId.Keyboard_Function, (5, 12) },
```

Comment:
```suggestion
        { LedId.Keyboard_Application, (5, 12) },
```

Reply:
should be function as seen here 
![image](https://user-images.githubusercontent.com/7173033/204252091-160cb0d5-db6a-4bc7-a6b2-ce87fdd81bfb.png)